local_map_filters:

  # 平滑部分区域
  - name: traversability_mean
    type: gridMapFilters/MeanInRadiusFilter
    params:
      input_layer: traversability
      output_layer: traversability_mean
      radius: 0.4 # in m.
      
  # Set upper threshold on traversability.
  - name: traversability_upper_threshold
    type: gridMapFilters/ThresholdFilter
    params:
      condition_layer: traversability_mean
      output_layer: traversability
      upper_threshold: 1.0
      set_to: 1.0 # Other uses: .nan, .inf